core/src/hir/attrs.rs

```diff
@@ -548,7 +575,7 @@ impl Attrs {
 /// ```ignore
 /// struct Sample {}
 /// impl Sample {
-///     #[diplomat::attr(*, constructor)]
+///     #[diplomat::attr(supports = constructors, constructor)]
```

Comment:
actually using these will be niche, so revert this?

Reply:
yeah my bad